README.md

```diff
@@ -45,7 +46,23 @@ source ~/rds/rds-ukaea-mast-sPGbyCAPsJI/uda-ssl.sh
 
 You should now be able to run the following commands.
 
-## Local Ingestion
+### Submitting runs on CSD3
+
+1. First submit a job to collect all the metadata:
+
+```sh
+sbatch ./jobs/metadata.csd3.slurm.sh
+```
```

Comment:
if you want it more foolproof perhaps add something to `cd ..` out of the mastcodes folder, the above step puts the user in mastcodes and not in the directory this step requires